lib/timescale/hyperfunctions.ex

```diff
@@ -5,6 +5,29 @@ defmodule Timescale.Hyperfunctions do
   IT systems, marketing analytics, user behavior, financial metrics, and cryptocurrency.
 
   [Documentation](https://docs.timescale.com/api/latest/hyperfunctions/)
+
+  ## Approximate Row Count
+
+  [Documentation](https://docs.timescale.com/api/latest/hyperfunctions/approximate_row_count/)
+
+  Timescale offers a hyperfunction, `approximate_row_count`, for retrieving the approximate row count of hypertables.
+  This hyperfunction may be useful for tables with a large number of rows, where the exact number is not needed.
+
+  Rather than supporting it directly, we recommend that you use the following code if this is a useful feature to your
```

Comment:
We can probably add a reason why it is not supported so people aren't left wondering.

Reply:
good idea